deploy/gcp/tidbclusters.tf

```diff
@@ -26,7 +21,7 @@ module "default-tidb-cluster" {
   gke_cluster_name           = module.tidb-operator.gke_cluster_name
```

Comment:
in my testing these output variables are not available too, I guess we need to use variables in root module.

Reply:
it seems we should follow this pattern: https://www.terraform.io/docs/modules/composition.html#dependency-inversion